arm-compute/2016-09-30/containerService.json

```diff
@@ -0,0 +1,561 @@
+              "$ref": "#/definitions/Resource"
+            }
+          ],
+          "description": "Container service"
+        },
+        "ContainerServiceListResult": {
+          "properties": {
+            "value": {
+              "type": "array",
+              "items": {
+                 "$ref": "#/definitions/ContainerService"
+              },
+              "description": "Gets or sets the list of container services."
+            }
+          },
+          "description": "The List Container Service operation response"
+        },
+        "ContainerServiceProperties": {
+          "properties": {
+            "provisioningState": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Gets the provisioning state, which only appears in the response."
+            },
+            "customProfile": {
+              "$ref": "#/definitions/ContainerServiceCustomProfile",
+              "description": "Properties for customizing this cluster"
+            },
+            "orchestratorProfile": {
+              "$ref": "#/definitions/ContainerServiceOrchestratorProfile",
+              "description": "Properties of orchestrator"
+            },
+            "masterProfile": {
+              "$ref": "#/definitions/ContainerServiceMasterProfile",
+              "description": "Properties of master agents"
+            },
+            "agentPoolProfiles": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ContainerServiceAgentPoolProfile"
+              },
+              "description": "Properties of agent pools"
+            },
+            "windowsProfile": {
+              "$ref": "#/definitions/ContainerServiceWindowsProfile",
+              "description": "Properties of Windows VMs"
+            },
+            "linuxProfile": {
+              "$ref": "#/definitions/ContainerServiceLinuxProfile",
+              "description": "Properties for Linux VMs"
+            },
+            "diagnosticsProfile": {
+              "$ref": "#/definitions/ContainerServiceDiagnosticsProfile",
+              "description": "Properties for Diagnostic Agent"
+            },
+            "servicePrincipalProfile": {
+              "$ref": "#/definitions/ContainerServiceServicePrincipalProfile",
+              "description": "Properties for the cluster service principal"
+            }
+          },
+          "required": [
+            "masterProfile",
+            "agentPoolProfiles",
+            "linuxProfile"
+          ],
+          "description": "Properties of container service"
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+          "name": "subscriptionId",
+          "in": "path",
+          "required": true,
+          "type": "string",
+          "description": "Gets subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."

```

Comment:
Please stop writing hydra-specs and converting them to swagger. Please write swagger specs directly.